guides/NEW_LIBRARY.md

```diff
@@ -0,0 +1,38 @@
+Creating a new library
+======================
+
+The following steps will guide you through adding a new **library** to our system that is built by the CI system and deployed to the live environment.
+
+**NOTE:** If you're creating a new app go to [new app](NEW_APP.md)
+
+Prerequisites
+-------------
+
+To follow this guide you will need:
+
+* A Github account with access to the ONSdigital organisation
+
+* A login for the concourse CI environment
+
+* [dp-cli](https://github.com/ONSdigital/dp-cli)
+
+* Git:
+
+  `$ brew install git`
+
+* Fly CLI (can be downloaded from the links in the corner of the concourse UI)
```

Comment:
Possibly link to concourse UI here:

".... links in the bottom right corner of the [concourse UI](https://concourse.onsdigital.co.uk/)"